mobile/Screen/EventScreen.js

```diff
@@ -23,109 +25,103 @@ const styles = StyleSheet.create({
-    website: 'studio526.com',
-    url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/ea/Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg/1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg',
-  },
-  {
-    id: 2,
-    title: 'Shelter',
-    day: 'Tue, Thu',
-    time: '9am-5pm',
-    location: '4102 Daisy Rd, LA',
-    summary: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod temporincididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam aquis nostrud exercitation ullamcolaboris nisi ut aliquip ex ea commodo consequat.',
-    number: '(123) 345 5678',
-    email: 'studio526@gmail.com',
-    website: 'studio526.com',
-    url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/ea/Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg/1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg',
-  },
-  {
-    id: 3,
-    title: 'Mission',
-    day: 'Mon, Wed, Fri',
-    time: '9-10am',
-    location: '4123 Blue Rd, LA',
-    summary: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod temporincididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam aquis nostrud exercitation ullamcolaboris nisi ut aliquip ex ea commodo consequat.',
-    number: '(123) 345 5678',
-    email: 'studio526@gmail.com',
-    website: 'studio526.com',
-    url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/ea/Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg/1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg',
-  },
-
-];
-
 function EventScreen({ navigation }) {
   const [showCalendar, setShowCalendar] = useState(false);
   const [selectedDate, setSelectedDate] = useState(null);
+  const [allEvents, setAllEvents] = useState([]);
+  const [filteredEvents, setFilteredEvents] = useState([]);
+  const [fontsLoaded] = useFonts({
+    Montserrat: Montserrat_400Regular,
+    MontserratSemiBold: Montserrat_600SemiBold,
+    MontserratBold: Montserrat_700Bold,
+    MontserratMedium: Montserrat_500Medium,
+  });
+
+  const eventThumbnails = {
+    'art & community': require('../assets/eventThumbnails/artCommunity.png'),
+    exhibit: require('../assets/eventThumbnails/exhibit.png'),
+    film: require('../assets/eventThumbnails/film.png'),
+    music: require('../assets/eventThumbnails/music.png'),
+    performance: require('../assets/eventThumbnails/performance.png'),
+    'spoken word': require('../assets/eventThumbnails/spokenWord.png'),
+    miscellaneous: require('../assets/eventThumbnails/miscellaneous.png'),
+    'visual art': require('../assets/eventThumbnails/visualArt.png'),
+  };
+
+  const getAllEvents = async () => {
+    try {
+      const result = await axios.get(`${URL}/event/get`);
+      setAllEvents(result.data);
+      setFilteredEvents(result.data);
+      return result.data;
+    } catch (err) {
+      console.error(err);
+      return err;
+    }
+  };
+
+  useEffect(() => {
+    getAllEvents();
+  }, []);
 
   const handleButtonPress = () => {
```

Comment:
Hi-fis seem to indicate the calendar should show all the time? do UK if that's still a thing